src/components/DpIcon/util/iconConfig.ts

```diff
@@ -0,0 +1,184 @@
+import {
+  PhArrowDown,
+  PhArrowLeft,
+  PhArrowRight,
+  PhArrowsClockwise,
+  PhArrowUp,
+  PhCaretDown,
+  PhCaretLeft,
+  PhCaretRight,
+  PhCaretUp,
+  PhCaretDoubleDown,
+  PhCaretDoubleUp,
+  PhCheck,
+  PhCheckCircle,
+  PhCircleNotch,
+  PhClock,
+  PhCopy,
+  PhCornersIn,
+  PhCornersOut,
+  PhClockCounterClockwise,
+  PhDownloadSimple,
+  PhDotsSixVertical,
+  PhEnvelopeSimple,
+  PhFile,
+  PhGearSix,
+  PhHighlighter,
+  PhHourglass,
+  PhInfo,
+  PhLockSimple,
+  PhLockSimpleOpen,
+  PhMagnifyingGlass,
+  PhPencilSimple,
+  PhPhone,
+  PhPlus,
+  PhRobot,
+  PhQuestion,
+  PhSignOut,
+  PhTag,
+  PhTrash,
+  PhUser,
+  PhUsersThree,
+  PhWarning,
+  PhWarningDiamond,
+  PhX,
+  PhXCircle
+} from '@phosphor-icons/vue'
+import {
+  AliasedPhosphorIconName,
+  IconAlias,
+  IconName,
+  IconProportion,
+  IconSize,
+  PhosphorIconName} from '../../../../types'
+import type { Component } from 'vue'
+
+// ICONS
+
+// icon names that are used and already correspond to the phosphor icon name
+const mappedIconNames: Record<PhosphorIconName, Component> = {
+  'arrow-down': PhArrowDown,
+  'arrow-left': PhArrowLeft,
+  'arrow-right': PhArrowRight,
+  'arrow-up': PhArrowUp,
+  'caret-down': PhCaretDown,
+  'caret-left': PhCaretLeft,
+  'caret-right': PhCaretRight,
+  'caret-up': PhCaretUp,
+  'caret-double-down': PhCaretDoubleDown,
+  'caret-double-up': PhCaretDoubleUp,
+  'check': PhCheck,
+  'check-circle': PhCheckCircle,
+  'clock': PhClock,
+  'copy': PhCopy,
+  'file': PhFile,
+  'highlighter': PhHighlighter,
+  'hourglass': PhHourglass,
+  'info': PhInfo,
+  'plus': PhPlus,
+  'phone': PhPhone,
+  'question': PhQuestion,
+  'sign-out': PhSignOut,
+  'tag': PhTag,
+  'user': PhUser,
+  'users-three': PhUsersThree,
+  'warning': PhWarning,
+  'x': PhX,
+  'x-circle': PhXCircle
+}
+
+// icon names that are not used yet but should replace the aliases
+const mappedAliasedIconNames: Record<AliasedPhosphorIconName, Component> = {
```

Comment:
Why are you passing `AliasedPhosphorIconName` here, but in line 59 `PhosphorIconName`? I see no difference in the two objects, both consist of a string corresponding to a PhosphorIcon component.

Reply:
The `AliasedPhosphorIconName` type contains the names that are aliases for phosphor icon names, the `PhosphorIconName` type contains names that are identical to phosphor icon names. My idea was to separate these so removing the aliases will be easier later.  I found it rather time consuming to understand which icon name corresponds to which phosphor icon, but maybe it's easier now that we map the names to phosphor icon components directly.